spring-rabbit/src/test/java/org/springframework/amqp/rabbit/listener/BlockingQueueConsumerTests.java

```diff
@@ -240,11 +243,14 @@ private void testRequeueOrNotGuts(Exception ex, boolean expectedRequeue,
 	public void testAlwaysCancelAutoRecoverConsumer() throws IOException {
 		ConnectionFactory connectionFactory = mock(ConnectionFactory.class);
 		Connection connection = mock(Connection.class);
-		Channel channel = mock(AutorecoveringChannel.class);
+		ChannelProxy channel = mock(ChannelProxy.class);
+		Channel rabbitChannel = mock(AutorecoveringChannel.class);
+		when(channel.getTargetChannel()).thenReturn(rabbitChannel);
 
 		when(connectionFactory.createConnection()).thenReturn(connection);
 		when(connection.createChannel(anyBoolean())).thenReturn(channel);
-		when(channel.isOpen()).thenReturn(true);
+		final AtomicBoolean isOpen = new AtomicBoolean(true);
```

Comment:
Addressed via https://github.com/spring-projects/spring-amqp/commit/b238d6a6012d74920d1959fc581effd0d21a2120 and one more especially for `master`: https://github.com/spring-projects/spring-amqp/commit/d7e702a765bbd18b2a64c61297795e1cbe504e69